distributions/demo-server/build.gradle

```diff
@@ -40,6 +55,23 @@ dependencies {
 
 addCloudExtensionDependency(project)
 
+task createSecretsFile() {
+
+    doLast {
```

Comment:
nice. Tagging #236 to track that secrets.csv is being moved into the gradle script in this PR